test_scripts/WebEngine/WebEngineProjection/025_OnExitApplication_RESOURCE_CONSTRAINT_multiple_apps_on_different_connections.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Processing of the OnExitApplication notification with reason RESOURCE_CONSTRAINT from HMI
+--  (multiple applications on different connections)
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. App1 is registered on mobile connection 1
+-- 3. App2 is registered on web engine connection 2
+-- 4. App2 is activated
+--
+-- Steps:
+-- 1. HMI sends BC.OnExitApplication with reason: "RESOURCE_CONSTRAINT" related to App2 to SDL
+-- SDL does:
+--  - unregister App2 and send OnAppInterfaceUnregistered notification with reason: "RESOURCE_CONSTRAINT" to it
+--  - send BasicCommunication.OnAppUnregistered notification related to App2 with unexpectedDisconnect: false
+--  - close mobile connection 2
```

Comment:
@aderiabin if `3. App2 is registered on web engine connection 2`
```suggestion
--  - close web engine connection 2
```

Reply:
@LitvinenkoIra  Done(see d8172a5 )